src/main/kotlin/com/hyperdevs/poeditor/gradle/PoEditorStringsImporter.kt

```diff
@@ -80,18 +82,41 @@ object PoEditorStringsImporter {
                               defaultLang: String,
                               resDirPath: String,
                               tags: List<String>?,
-                              languageValuesOverridePathMap: Map<String, String>?) {
+                              languageValuesOverridePathMap: Map<String, String>?,
```

Comment:
I'm going to do a quick change here so we don't pass null values here, since the properties already return empty values, just in case it conflicts with the changes here.

Reply:
OK.
Btw I often have trouble with detekt. Like now when merging master into my branch.
When there's a problem and the commit/merge commit is blocked, the error output is hard to read, it's all in one line 🙂
Maybe it's not correctly set up on my machine, didn't spend time on this.

It may be a good idea to add a setup readme for contributors. (I was also wondering if you prefer merging or rebasing from master etc)

Reply:
Here's some output. In this case it's actually pretty short, but normally it also mentions some formatting/complexity warnings etc.

`Couldn't check the working tree for unmerged files because of an error. tput: No value for $TERM and no -T specified tput: No value for $TERM and no -T specified tput: No value for $TERM and no -T specified tput: No value for $TERM and no -T specified Running detekt > Task :detekt UP-TO-DATE Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0. Use '--warning-mode all' to show the individual deprecation warnings. See https://docs.gradle.org/6.9/userguide/command_line_interface.html#sec:command_line_warnings BUILD SUCCESSFUL in 814ms 1 actionable task: 1 up-to-date Your code is properly formatted! could not read log file '/Users/.../poeditor-android-gradle-plugin/.git/MERGE_MSG': No such file or directory`

Reply:
Hmmm, OK! I'll take a look into that. 
I opened #41 to track the error that Detekt outputs, and #42 to track contribution guidelines (since it has been a project the team has developed mostly internally until recently we never thought about proper contribution guidelines, to be honest 😅).

Feel free to provide any info that could be useful there.